src/components/panel/panel.jsx

```diff
@@ -12,9 +11,21 @@ var Panel = React.createClass({
   displayName: 'Panel',
 
   propTypes: {
-    title: React.PropTypes.string, // Panel title
-    displayModes: React.PropTypes.array,
-    actions: React.PropTypes.array
+    title: PropTypes.node,
+    titleActions: PropTypes.node,
+    headerActions: PropTypes.node,
+    displayModes: PropTypes.array,
+    defaultDisplayModeId: PropTypes.string,
+    // TODO: Maybe replace actions with PropTypes.node
+    actions: PropTypes.array
```

Comment:
You really need to document format of actions as there are many possibilities otherwise component might be hard to use.